src/huggingface_hub/file_download.py

```diff
@@ -414,18 +414,38 @@ def http_get(
     url: str,
     temp_file: BinaryIO,
     *,
-    proxies=None,
+    proxies: Optional[Dict] = None,
     resume_size: float = 0,
     headers: Optional[Dict[str, str]] = None,
     expected_size: Optional[int] = None,
+    filename: Optional[str] = None,
```

Comment:
maybe display_filename to emphasize that it's not the filename where the file is going to be saved at?

Reply:
Good idea. Removed in https://github.com/huggingface/huggingface_hub/pull/1954/commits/6724e6cdec39988d1c5bbdb5e2735e07b95908da.